diff --git a/src/Castle.Core/Compatibility/TypeBuilderExtensions.cs b/src/Castle.Core/Compatibility/TypeBuilderExtensions.cs
index 5946120ae..9bf6e3aeb 100644
--- src/Castle.Core/Compatibility/TypeBuilderExtensions.cs
+++ src/Castle.Core/Compatibility/TypeBuilderExtensions.cs
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#if FEATURE_LEGACY_REFLECTION_API
+//#if FEATURE_LEGACY_REFLECTION_API
 
 namespace System.Reflection
 {
@@ -35,4 +35,4 @@ namespace System.Reflection
 	}
 }
 
-#endif
+//#endif